aws-mfa

```diff
@@ -23,38 +23,43 @@ logger.addHandler(stdout_handler)
 logger.setLevel(logging.DEBUG)
 
 AWS_CREDS_PATH = '%s/.aws/credentials' % (os.path.expanduser('~'),)
-STS_DEFAULT = 900
+STS_DEFAULT = 43200  # in seconds. AWS default
 
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--device',
-                        required=False,
-                        metavar='arn:aws:iam::123456788990:mfa/dudeman',
-                        help="The MFA Device ARN. This value can also be "
-                        "provided via the environment variable 'MFA_DEVICE'")
-    parser.add_argument('--duration',
-                        type=int,
-                        help="The duration, in seconds, indicating how long "
-                        "the temporary credentials should be valid. The "
-                        "minimum is 900 seconds (15 minutes) and the maximum "
-                        "is 3600 seconds (1 hour). This value can also be "
-                        "provided via the environment variable "
-                        "'MFA_STS_DURATION'.")
-    parser.add_argument('--profile',
-                        help="If using profiles, specify the name here. The "
-                        "default profile name is 'default'",
-                        required=False)
-    parser.add_argument('--assume-role',
-                        metavar='arn:aws:iam::123456788990:role/RoleName',
-                        help="The ARN of the AWS IAM Role you would like to "
-                        "assume, if specified. This value can also be provided"
-                        " via the environment variable 'MFA_ASSUME_ROLE'",
-                        required=False)
-    parser.add_argument('--role-session-name',
-                        help="Friendly session name required when using "
-                        "--assume-role",
-                        required=False)
+    parser.add_argument(
+        '--device',
+        required=False,
+        metavar='arn:aws:iam::123456788990:mfa/dudeman',
+        help="""The MFA Device ARN. This value can also be provided via the
+        environment variable 'MFA_DEVICE'""")
+    parser.add_argument(
+        '--duration',
+        type=int,
+        help="""The duration, in seconds, that the credentials should remain
+        valid. Acceptable durations for IAM user sessions range from 900
+        seconds (15 minutes) to 129600 seconds (36 hours), with 43200 seconds
+        (12 hours) as the default. Sessions for AWS account owners are
+        restricted to a maximum of 3600 seconds (one hour). If the duration
+        is longer than one hour, the session for AWS account owners defaults
+        to one hour.""")
```

Comment:
Good point. For reference, the doc for assume-role is here:
http://docs.aws.amazon.com/cli/latest/reference/sts/assume-role.html
```
--duration-seconds (integer)
The duration, in seconds, of the role session. The value can range from 900 seconds (15 minutes) to 3600 seconds (1 hour). By default, the value is set to 3600 seconds.
```